pkgs/development/python-modules/pyqt/4.x.nix

```diff
@@ -17,9 +17,15 @@ in mkPythonDerivation {
     rm -rf "$out/nix-support"
 
     export PYTHONPATH=$PYTHONPATH:$out/lib/${python.libPrefix}/site-packages
+    ${stdenv.lib.optionalString stdenv.isDarwin ''
```

Comment:
Whoops. I'm sorry. That's why you should probably not push untested code. The I suppose I will re-add stdenv then, because I wouldn't know how to check for `isDarwin` with `lib` only.